sdb/__init__.py

```diff
@@ -191,3 +191,37 @@ def invoke(prog: drgn.Program, first_input: Iterable[drgn.Object],
         if shell_cmd is not None:
             sys.stdout = old_stdout
             shell_proc.wait()
+
+
+def type_equals(a: drgn.Type, b: drgn.Type) -> bool:
```

Comment:
From our conversation:
[1] Qualifiers
[2] C++ class
[3] Incomplete types (trippy size 0) -  keep in mind that `void` is always incomplete
[4] Function types and their typedefs
[5] Can still keep this function and have it call the new one that returns the canonical names and oes string comparisons.

Reply:
I think that these are all addressed by the new version (either by fixing or by commenting).